backend/src/main/java/ro/utcn/sd/agui/a1/controller/AnswersController.java

```diff
@@ -0,0 +1,58 @@
+package ro.utcn.sd.agui.a1.controller;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+import ro.utcn.sd.agui.a1.dto.AnswerDTO;
+import ro.utcn.sd.agui.a1.entity.User;
+import ro.utcn.sd.agui.a1.service.AnswerManagementService;
+import ro.utcn.sd.agui.a1.service.AnswerVoteManagementService;
+import ro.utcn.sd.agui.a1.service.UserDetailsManagementService;
+
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+public class AnswersController {
+    private final AnswerManagementService answerManagementService;
+    private final UserDetailsManagementService userDetailsManagementService;
+    private final AnswerVoteManagementService answerVoteManagementService;
+
+    @GetMapping("/question/{questionId}/answers")
+    public List<AnswerDTO> listAllAnswersPerQuestion(@PathVariable("questionId") int questionId) {
+        return answerManagementService.listAllAnswersPerQuestion(questionId);
+    }
+
+    @GetMapping("/question/{questionId}/answers-by-score")
+    public List<AnswerDTO> listAllAnswersPerQuestionByScore(@PathVariable("questionId") int questionId) {
```

Comment:
Small hint: you could use `@RequestParam`s for passing the attribute by which to do the sorting (instead of having a dedicated endpoint for a specific sort). Also, you can omit the name inside the `@PathVariable` if the name is the same as the name of the method parameter.